crates/rpc-types-eth/src/transaction/mod.rs

```diff
@@ -103,7 +106,10 @@ pub struct Transaction {
     )]
     #[doc(alias = "tx_type")]
     pub transaction_type: Option<u8>,
-
+    /// The authorization list is a list of tuples that store the address to code which the signer
+    /// desires to execute in the context of their EOA.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub authorization_list: Option<Vec<Authorization>>,
```

Comment:
I believe we actually want the SignedAuthoriazion here because we also need the signature

Reply:
Updated

Reply:
Seems fixed!